### ISSUE TITLE
Seperate cooling mode from heating mode

### ISSUE THREAD

**Issue by samdave123** [RainbowMiner/RainbowMiner#748]
I've noticed an issue with the cooling mode where it'll favour lower wattage algorithms rather than more efficient algorithms.

Proposed solution:
Three seperate modes
- Heating mode
- Cooling mode
- Normal profit-switching (both cooling and heating mode disabled)

The heating mode is working extremely well now, so I suggest that remains unchanged, but just limit the value range to remove the cooling.

The cooling mode could use a formula like this:
=(sats/power consumption) * (X*profit)

When X is 0, the most efficient algorithm will be chosen, and when X is above 0 RBM will choose something between peak efficiency and peak profitability.

**Comment by Yamanipanuchi** [comment]
I agree there's some validity here, as when I use to lower power mode I am looking to keep the heat down. I do notice that you still need to have a power price value. Which greatly sways the choices made. My situation stands that the power cost is not something I need to factor, so I usually leave it at $.00 which effectively eliminate the heat/cool running modes.

Might not actually help, But my two cents.

**Comment by samdave123** [comment]
> 
> 
> I agree there's some validity here, as when I use to lower power mode I am looking to keep the heat down. I do notice that you still need to have a power price value. Which greatly sways the choices made. My situation stands that the power cost is not something I need to factor, so I usually leave it at $.00 which effectively eliminate the heat/cool running modes.
> 
> Might not actually help, But my two cents.

I think this proposed rework should solve that issue for you. The current calculation multiplies/divides powercost by a value you choose. The calculation I mentioned above is a multiplication of profit and efficiency, so power cost doesn't even need to be factored.

**Comment by samdave123** [comment]
Currently we're experiencing a mini heatwave in UK. The current controls over power efficiency are a bit awkward to use because it is increasing the power cost in the profit-switching.

I need an option to enable the profit-switcher to switch directly to the algo with the highest sat/W. Perhaps an additional checkbox which then overrides the heat control?

**Comment by SuperBartimus** [comment]
I like this idea too.  I'm in the situation that I need to either lower my overall wattage usage due to electricity costs or in the Texas Gulf Coast region, need to keep the heat down.
